src/content/en/updates/2019/08/barcode-detection.md

```diff
@@ -0,0 +1,140 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The Barcode Detection API allows for barcodes to be detected in images.
+
+{# wf_updated_on: 2019-08-06 #}
+{# wf_published_on: 2019-08-01 #}
+{# wf_tags: capabilities,shape-detection,progressive-web-apps,webapp #}
+{# wf_featured_image: /web/updates/images/generic/timeline.png #}
+{# wf_featured_snippet: The Barcode Detection API allows for barcodes to be detected in images. #}
+{# wf_blink_components: Blink>ShapeDetection #}
+
+# Shipping the Barcode Detection Part of the Shape Detection API {: .page-title}
+
+{% include "web/_shared/contributors/thomassteiner.html" %}
+
+<div class="clearfix"></div>
+
+Success: We were working on the specification for this API as part of the
+[capabilities project](/web/updates/capabilities). Following a successful Origin Trial,
+barcode detection has now launched.
+
+You may remember [my post from January](../01/shape-detection)
+on the [Shape Detection API](https://wicg.github.io/shape-detection-api),
+an API that defines three interfaces: `BarcodeDetector`, `FaceDetector`,
+and `TextDetector`. I'm now happy to share that following a successful Origin Trial,
+we're shipping the `BarcodeDetector` part of the Shape Detection API in Chrome&nbsp;78.
+The `FaceDetector` and the `TextDetector` interface still need some more fine-tuning.
+
+### Working With the `BarcodeDetector` {: #barcodedetector}
+
+Let's briefly recall how working with the `BarcodeDetector` looks like:
```

Comment:
"Let's briefly recall how working with the `BarcodeDetector` looks like"

This sentence is more complicated than it needs to be. Maybe:

"Recall how `BarcodeDetector` works:"

Reply:
Done.